tests/libtest.sh

```diff
@@ -282,10 +296,13 @@ EOF
 
     case $bootmode in
         "syslinux")
-	    setup_os_boot_syslinux
+        setup_os_boot_syslinux
             ;;
```

Comment:
Does OSTree repository has policy regarding spaces vs tabs? When I see tabs around lines that I am touching I feel like I have to convert everything to spaces :)